index.js

```diff
@@ -1,26 +1,118 @@
 // 1.Fibonacci
 
-// function fibo(n) {
-//   return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
-// }
-// fibo(5);
-// fibo(6);
-// fibo(7);
+function fibo(n) {
+  return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
+}
+fibo(5);
+fibo(6);
+fibo(7);
 
 // 2. Implementing with reduce
 
 //concat;
+let arr = [1, 2];
+let arr1 = [3, 4];
+
+const concatFn = (array, newEl) =>
```

Comment:
you are mutating state, please note that `let arr = [1, 2];` is changed, please don't change the variable